common/src/main/java/noobanidus/mods/lootr/common/api/ILootrAPI.java

```diff
@@ -166,10 +167,6 @@ default boolean isDefaultTextures() {
 
   boolean shouldCheckWorldBorder();
 
-  int getMaximumAge();
-
-  boolean hasExpired(long time);
-
```

Comment:
the "age" concept can lose track of chests that are near unloaded chunks for a long time. instead of this, I added logic to check if the chunk the chest is in has unloaded, which clears it from the list faster and doesn't miss chests.